src/Goerli-DssSpell.sol

```diff
@@ -72,6 +85,31 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
         // Includes changes from the DssSpellCollateralAction
         // onboardCollaterals();
         // offboardCollaterals();
+
+        // Starknet Bridge Upgrade
+        // https://github.com/makerdao/starknet-dai-bridge#upgrades
+        
+        // close current bridge
+        address currentStarknetDAIBridge = DssExecLib.getChangelogAddress("STARKNET_DAI_BRIDGE");
+        StarknetBridgeLike(currentStarknetDAIBridge).close();
+
+        // approve new bridge
+        address NEW_STARKNET_DAI_BRIDGE = 0xaB00D7EE6cFE37cCCAd006cEC4Db6253D7ED3a22;
+        address starknetEscrow = DssExecLib.getChangelogAddress("STARKNET_ESCROW");
+        address dai = DssExecLib.getChangelogAddress("MCD_DAI");
+        StarknetEscrowLike(starknetEscrow).approve(dai, NEW_STARKNET_DAI_BRIDGE, type(uint).max);
+
+        // relay l2 spell
+        // See: https://goerli.voyager.online/contract/0x04363a4e51a9d2eaccef7a7ef5f0c8872f8183db2179802c0907f547c87864fc#code
+        address starknetGovRelay = DssExecLib.getChangelogAddress("STARKNET_GOV_RELAY");
+        uint256 L2_FEE_SPELL = 0x04363a4e51a9d2eaccef7a7ef5f0c8872f8183db2179802c0907f547c87864fc;
+        StarknetGovRelayLike(starknetGovRelay).relay(L2_FEE_SPELL);
+
+        // ChangeLog
+        DssExecLib.setChangelogAddress("STARKNET_DAI_BRIDGE", NEW_STARKNET_DAI_BRIDGE);
+        DssExecLib.setChangelogAddress("STARKNET_DAI_BRIDGE_LEGACY", currentStarknetDAIBridge);
+
+        DssExecLib.setChangelogVersion("1.14.3");
```

Comment:
```suggestion
```

This is good and matches the current version we have in the spell from the rETH onboarding, so would go with our tests. 

Just making a note of this though since the rETH code will already bump the version, I'd say we can remove it here.

Reply:
Removed